### ISSUE TITLE
fix: Missing space before function parentheses ?

### ISSUE THREAD

**Issue by koolay** [beautify-web/js-beautify#1077]
Eslint standard required:

`Missing space before function parentheses`

beautify result:

```
    export default {
        data() {
            return {
                input: ''
            }
        }
    }

```

expected:

```
    export default {
        data () { // this row
            return {
                input: ''
            }
        }
    }

```

**Comment by bitwiseman** [comment]
This would be a difference for the rest of the function declarations.  
`function bar() {}` not `function bar () {}`.

**Comment by koolay** [comment]
has fixed ?

**Comment by bitwiseman** [comment]
No.  This is an enhancement that would require an additional setting.  I encourage you to submit a pull request with tests.

**Comment by cody1991** [comment]
using vue-cli eslint and also has this error:

```Missing space before function parentheses```

 function bar() {} not function bar () {}.

I can just change 

```function bar()```

to 

```let bar = () => {}```

waiting for fix this ~

**Comment by budapeisidajiudian** [comment]
A function inside an object like this:
``` javascript
var myObj = {
    foo () {
        // something
    }
}
```
I can't use the arrow function
waiting for fix this or giving an option

**Comment by jernejrostan** [comment]
Anything new with this?

**Comment by bahatron** [comment]
it would definetly be nice to have this implemented

**Comment by tahaerden** [comment]
Having the same issue as I use js-standard formatting.
Linter package I use: https://github.com/ricardofbarros/linter-js-standard
Actual error: Missing space before function parantheses. (space-before-function-paren)

**Comment by bitwiseman** [comment]
There is now a `space_after_named_function` setting (see #608).  

However, in the statement `a = {  myfunc() {} };`, "myfunc" isn't recognized as a function expression.

**Comment by bitwiseman** [comment]
Fixed in #1425.
